geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/conf/GeoMesaProperties.scala

```diff
@@ -13,27 +13,81 @@ import java.util.Properties
 import com.typesafe.scalalogging.LazyLogging
 
 object GeoMesaProperties extends LazyLogging {
-
   private val EmbeddedFile = "/org/locationtech/geomesa/geomesa.properties"
-
   private val props: Properties = {
     val resource = getClass.getResourceAsStream(EmbeddedFile)
+    val props = new Properties
     if (resource == null) {
       logger.warn(s"Couldn't load $EmbeddedFile")
-      new Properties
+      props
     } else {
-      val p = new Properties
       try {
-        p.load(resource)
+        props.load(resource)
       } finally {
         resource.close()
       }
-      p
+      props
     }
   }
 
   val ProjectVersion = props.getProperty("geomesa.project.version")
   val BuildDate      = props.getProperty("geomesa.build.date")
   val GitCommit      = props.getProperty("geomesa.build.commit.id")
   val GitBranch      = props.getProperty("geomesa.build.branch")
+
+  val GEOMESA_CONFIG_FILE      = ConfigLoader.GEOMESA_CONFIG_FILE
+  val GEOMESA_CONFIG_FILE_NAME = ConfigLoader.GEOMESA_CONFIG_FILE_NAME
+
+  def GEOMESA_TOOLS_ACCUMULO_SITE_XML: String = {
```

Comment:
Any thoughts on how we'd do that? I'm unsure how, since the only copy atm is the .xml.template. Unless we somehow scan that file (?) or something at compile time and build the xml.template.